src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -659,6 +657,42 @@ static void AnimateRootView(CGRect rect)
 			ContainerView.Frame = rect;
 	}
 
+	static void ApplyContentInset(UIScrollView? scrolledView, UIScrollView? lastScrollView)
+	{
+		if (scrolledView is null || lastScrollView is null || ContainerView is null)
+			return;
+
+		var keyboardIntersect = CGRect.Intersect(KeyboardFrame, scrolledView.Frame);
+		nfloat movedContainerDistance = 0;
```

Comment:
This variable appears to be not used (besides being set once).

Reply:
Wait, now I see what you wanted to achieve, there's probably a mistake on line 671, it should be:
```cs
if (TopViewBeginOrigin != InvalidPoint)
{
    movedContainerDistance = StartingContainerViewFrame!.Value.Y - ContainerView.Frame.Y;
}
```

and then below

```cs
var bottomInset = keyboardIntersect.Height - movedContainerDistance;
```

Reply:
I think this is actually needed, but unfortunately I don't remember the use case for this.
I will add a comment if anything comes to my mind. Thanks.